### PR TITLE
OCPBUGS-44014: Cancelling the file browser dialog causes TypeError crash

### PR DESCRIPTION
**Pull request by Mylanos**
When user closed the file browser, we attempted to read size from file that was undefined at that time, adding additional condition fixes this issue

### CODE REVIEW

**Review by jhadvig — APPROVED** [review]
/lgtm
/approve

**Comment by Mylanos** [comment]
Docs Approver:
/assign @opayne1
PX Approver:
/assign @reestr
QA Approver:
/assign @yapei

**Comment by yanpzhan** [comment]
@Mylanos This is a pr for main release bug, QEs usually test it after pr is merged, "px-approved","doc-approved","qe-approved" is not required, but we added them after saw your requirement in comment. I think we needn't add them for such kind of prs in future.

**Comment by Mylanos** [comment]
Sorry about that! @yanpzhan I just saw that the tide required either to add all of the labels I requested in my previous comment or remove the hold label, which I was unable to remove, so I assumed this is the only way how to pass the tide.

**Comment by yanpzhan** [comment]
Got it. Never mind, you did it.

**Pull request merged**
